### PR TITLE
Login and signup page added

### LINKED ISSUE #65: Will add the login and sign up page in the website and make some responsiveness changes
**Describe the solution you'd like**
I will add two pages login and signup pages with clear UI.

### PR DESCRIPTION
**Pull request by MAYANK816**
Added the login and signup page added improved some css for the pages please go through from the code.

### CODE REVIEW

**Comment by MAYANK816** [comment]
Please also use hacktober-fest accepted tag with my both issue and PR .

**Comment by MAYANK816** [comment]
@TheGuyDangerous Have you checked code if yes then could you please merge my request

**Comment by TheGuyDangerous** [comment]
alright, give me a minute

File: index.html [hunk 1]
```diff
@@ -67,6 +67,10 @@
                 class="u-active-grey-5 u-border-active-palette-1-base u-border-hover-palette-1-base u-button-style u-hover-grey-10 u-nav-link u-text-active-grey-90 u-text-grey-90 u-text-hover-grey-90"
                 href="index.html" style="padding: 10px 20px;">Home</a>
             </li>
+            <li class="u-nav-item"><a
+              class="u-active-grey-5 u-border-active-palette-1-base u-border-hover-palette-1-base u-button-style u-hover-grey-10 u-nav-link u-text-active-grey-90 u-text-grey-90 u-text-hover-grey-90"
+              href="login.html" style="padding: 10px 20px;">Login</a>
```
Comment (TheGuyDangerous) [inline index.html]: Create a separate login/Signup tappable text that is not in the top navbar .. but somewhere down, below "Go  to recipe" button.
Remove the button from the top navbar

**Review by TheGuyDangerous — CHANGES_REQUESTED** [review]
Fetch the latest changes made to index.html
there has been a lot of UI improvements that have not been synced.

File: index.html (hunk 1 above)
Comment (MAYANK816) [reply index.html]: image


image

you want things like these.

Comment (TheGuyDangerous) [reply index.html]: Yes Perfect , just add one little detail and we're good to go :)
image

 - Add a "Login/Sign Up" text down below the "Go to Recipes" button linking the login page.

Comment (MAYANK816) [reply index.html]: image

Will this work?

Comment (TheGuyDangerous) [reply index.html]: Make it a small tappable text , like the one found in your login.html file
eg.)
image

Comment (MAYANK816) [reply index.html]: The same text you want on home page that's in the picture above or any specific text 
image
or just like this?

Comment (MAYANK816) [reply index.html]: or we can write 
--> Want to order? Get Started 
and get started will be the hyperlink

Comment (MAYANK816) [reply index.html]: image

Comment (TheGuyDangerous) [reply index.html]: Perfect

Comment (MAYANK816) [reply index.html]: Pushed all the changes in my login_branch and I am resolving this conversation and thanks for helping me out sir

**Review by TheGuyDangerous — APPROVED** [review]
Look's Good , I'll merge it in a sec

**Pull request merged**